### PR TITLE
[DNM] test

### PR DESCRIPTION
**Pull request by Akasurde**
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: .zuul.d/sanity.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- job:
+    name: ansible-test-redhat-openshift-sanity
+    run: playbooks/downstream_sanity.yaml
+
+- project:
+    name: github.com/ansible-collections/community.okd
+    check:
```
Comment (goneri) [inline .zuul.d/sanity.yml:7]: replace  `check` with `3pci-check`.

File: .zuul.d/sanity.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      jobs:
+        - network-ee-sanity-tests:
+            voting: false
+    gate:
```
[the remark this answers is not in the document]
Comment (Akasurde) [reply .zuul.d/sanity.yml:11]: OK.

File: .zuul.d/sanity.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+- job:
```
Comment (goneri) [inline .zuul.d/sanity.yml:1]: The file extension is `.yml`, but it should be `.yaml`. See: https://zuul-ci.org/docs/zuul/reference/config.html#configuration-items

File: .zuul.d/sanity.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+- job:
+    name: ansible-test-redhat-openshift-sanity
+    run: playbooks/downstream_sanity.yaml
+    nodeset: fedora-35-1vcpu
+
+- project:
+    name: github.com/ansible-collections/community.okd
```
Comment (goneri) [inline .zuul.d/sanity.yaml:7]: The project name is: `github.com/openshift/community.okd` :-)

**Pull request closed without merging**